### PR TITLE
Red button mouse hover status has wrong color #1335

### PR DESCRIPTION
**Pull request by aroy114**
## Describe your changes
Changed the colour property of hover

## Issue Number
#1335 

## Please ensure all items are checked off before requesting a review:
- [x] I deployed the code locally.
- [x] I have performed a self-review of my code.
- [x] I have included the issue # in the PR.
- [x] I didn't use any hardcoded values (otherwise it will not scale, and will make it difficult to maintain consistency across the application).
- [x] I have labelled the PR correctly.
- [x] My PR is granular and targeted to one specific feature.
- [x] I took a screenshot or a video and attached this to PR if there is a UI change.

## Video

https://github.com/user-attachments/assets/dd18a367-c725-4014-bb1a-06d51014bf00

### CODE REVIEW

**Comment by gorkem-bwl** [comment]
Looks good @Anand-Royy - can you also please send a video / screenshot of how it looks for the dark theme as well?

**Comment by Anand-Royy** [comment]
@gorkem-bwl please check

https://github.com/user-attachments/assets/35ab24de-1423-4906-a005-06bb0141c120

**Comment by gorkem-bwl** [comment]
Looks good!

File: Client/src/Utils/Theme/constants.js [hunk 1]
```diff
@@ -49,12 +49,12 @@ const paletteColors = {
...
 	green800: "#1C4428",
 	green900: "#12261E",
-	red50: "#F9ECED",
+	red50: "#912018",
 	red100: "#FBD1D1",
 	red200: "#F04438",
 	red300: "#D32F2F",
 	red700: "#542426",
-	red800: "#301A1F",
+	red800: "#912018",
```
Comment (ajhollid) [inline Client/src/Utils/Theme/constants.js:57]: Why are red50 and red800 the same colors?

**Review by ajhollid — CHANGES_REQUESTED** [review]
red50 and red800 are the same colors, this is semantically incorrect even if the colors display properly :thinking: 

What is trying ot be achieved her by setting the two colors to be the same value?

**Review by ajhollid — APPROVED** [review]
Thanks, looks good and makes more sense now!

**Pull request merged**